perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/build.sh

```diff
@@ -53,14 +68,14 @@ export PYTHONPATH="./"
 echo Installing requirements..
 pip install --require-hashes -r bigquery/requirements.txt --user
 
-CONFIG_ID=$(eval "python3 -m bigquery.get_experiments_config --gcsfuse_flags '$GCSFUSE_FLAGS' --branch '$BRANCH' --end_date '$END_DATE' --config_name '$CONFIG_NAME'")
+CONFIG_ID=$(eval "python3 -m bigquery.get_experiments_config --gcsfuse_flags '$GCSFUSE_FLAGS' --config_file_flag '$CONFIG_FILE_STRING' --branch '$BRANCH' --end_date '$END_DATE' --config_name '$CONFIG_NAME'")
 START_TIME_BUILD=$(date +%s)
 
 # Upload data to the gsheet only when it runs through kokoro.
 UPLOAD_FLAGS=""
 if [ "${KOKORO_JOB_TYPE}" == "RELEASE" ] || [ "${KOKORO_JOB_TYPE}" == "CONTINUOUS_INTEGRATION" ] || [ "${KOKORO_JOB_TYPE}" == "PRESUBMIT_GITHUB" ] || [ "${KOKORO_JOB_TYPE}" == "SUB_JOB" ];
 then
-  UPLOAD_FLAGS="--upload_gs --upload_bq --config_id $CONFIG_ID --start_time_build $START_TIME_BUILD"
+  UPLOAD_FLAGS="--upload_bq --config_id $CONFIG_ID --start_time_build $START_TIME_BUILD"
```

Comment:
**FYI:**
Removed --upload_gs as It will override the results exported by [this](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh) script.

Also, four experiments, when they run together, will override the results of each other. We will raise separate tasks if we want to have results on the gsheet also.